Runtime/MeshCombiner.cs

```diff
@@ -516,6 +516,46 @@ private static Matrix4x4 ScaleMatrix(ref Matrix4x4 matrix, float scale)
                 m33 = matrix.m33 * scale
             };
         }
+
+        static bool CanReadMesh(Mesh mesh) {
```

Comment:
It's minor, but I would prefer if you could keep the same consistency as the rest of the code and use the private access modifier on these new members.